packages/ic-asset-certification/README.md

```diff
@@ -525,3 +527,242 @@ fn cbor_encode(value: &impl Serialize) -> Vec<u8> {
 
 add_certificate_header(&mut response, &witness, &expr_path);
 ```
+
+## Deleting assets
+
+Deleting assets is similar to (certifying them)[#inserting-assets-into-the-asset-router].
+
+Depending on the configuration provided to the [certify_assets](AssetRouter::certify_assets) function,
+multiple responses may be generated for the same asset. To ensure that all generated responses are deleted,
+the [delete_assets](AssetRouter::delete_assets) function accepts the same configuration.
```

Comment:
please extend the docs by describing what happens if the configuration passed to `delete_assets` does not match the one passed previously to `certify_asset`.

Reply:
Included, in the next commit thanks.